Getting-Started/Data/Data-Types/index.md

```diff
@@ -1,5 +1,5 @@
 # Data-Types
-*A Data Type defines the type of input for a property. So when adding a property (on Document Types, Media Types and Members) when selecting the Type you are selecting a Data Type. There are a number of preconfigured Data Types available in Umbraco and more can be added in the Developer section.*
+*A Data Type defines the type of input for a property. So when adding a property (on Document Types, Media Types and Members) when selecting the Type you are selecting a Data Type. There are a number of pre-configured Data Types available in Umbraco and more can be added in the Developer section.*
```

Comment:
Why change?  Preconfigure is a know word: https://en.oxforddictionaries.com/definition/preconfigure

Reply:
Reverted :)